dll/win32/framedyn/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ list(APPEND SOURCE
     main.c
     chstring.cpp
     provider.cpp
+	WbemTime.cpp
```

Comment:
Please follow the indenting style of the file (indent with spaces, not tabs)

Reply:
I've fixed.